Map/MapBase.gd

```diff
@@ -5,5 +5,4 @@ class_name MapBase
 ## Holds basic functionality for a map
 
 @export var rooms: Array = [RoomBase] ## 2D array of rooms on the map. this will be assigned when generating rooms
-@export var width: PackedInt32Array = [2,3,2]
-
+@export var floors_width: PackedInt32Array = [2,3,2] ## array for variable map width, each element is the width of a floor
```

Comment:
good comment
on the topic of map width, you should check my latest comment #6 so you can see the latest ideas I had for map requirement width (i think it should make it so that the map will properly work, I'm not sure for the end though)

Reply:
this can be freely changed, and in the future it will be added with more in depth algorithms. i've changed it to reflect it though.